uberenv.py

```diff
@@ -388,7 +416,7 @@ def clone_repo(self):
             clone_opts = ("-c http.sslVerify=false " 
                           if self.opts["ignore_ssl_errors"] else "")
 
-            clone_cmd =  "git {0} clone --single-branch --depth=1 -b {1} {2} vcpkg".format(clone_opts, vcpkg_branch,vcpkg_url)
+            clone_cmd =  "git {0} clone --single-branch -b {1} {2} vcpkg".format(clone_opts, vcpkg_branch,vcpkg_url)
```

Comment:
Note: `--depth=1` broke our ability to use a custom commit

Reply:
Git cloning should also be merged into a function between the spack and vcpkg... next PR

Reply:
#49